core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackServiceMediaPlayer.java

```diff
@@ -335,6 +336,13 @@ final synchronized void setPlayerStatus(@NonNull PlayerStatus newStatus, Playabl
         callback.statusChanged(new PSMPInfo(oldPlayerStatus, playerStatus, getPlayable()));
     }
 
+    public boolean isAudioChannelInUse() {
+        AudioManager audioManager = (AudioManager) context.getSystemService(Context.AUDIO_SERVICE);
+        int mode = audioManager.getMode();
+
+        return (mode != AudioManager.MODE_NORMAL);
```

Comment:
> How about trying " Play AntennaPod, unplug headphone, start Phone call, plug in headaphone - podcast should not start."

Hmm, when doing an actual phone call, AntennaPod does not start to play on my device, even without your PR. When executing the same steps with a music player, AntennaPod "steals" the audio focus. I think that's pretty unexpected too, so how about doing something like this?

```suggestion
        return (audioManager.getMode() != AudioManager.MODE_NORMAL || audioManager.isMusicActive());
```